client/src/components/form/Form.jsx

```diff
@@ -0,0 +1,231 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import TextInput from './TextInput';
+import NumberInput from './NumberInput';
+
+//Import of graphics
+import cardDate from '../../assets/card/card-date.png';
```

Comment:
No big deal, but I would recommend you to use svg's for the icons, not only because they are vector based,  scalable and less fuzzy, but also because they use less memory.  And you can import and "place" a svg just like a png.